.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
```

Comment:
Those changes increases sync all command's execution time by %75

Master branch timestamps of first and last logs (16 seconds)
```
11-May-20 11:10:40 [INFO ]  Remote connection is up
11-May-20 11:10:56 [INFO ]  ------------------
```

First and last log timestamps from this PR: (28 seconds)
```
11-May-20 11:08:33 [INFO ]  Remote connection is up
11-May-20 11:09:01 [INFO ]  ------------------
```

Reply:
Should be fixed now

Reply:
With git prune stuff commented out, first and last timestamps for sync all command:
```
12-May-20 11:48:34 [INFO ]  Remote connection is up
12-May-20 11:49:02 [INFO ]  ------------------
```

It still takes 28 seconds. Since that PR is just about bug fixes & improvements without functionality change, I wonder where is that %75 execution time increase coming from.

Reply:
To make it more clear, excluding prune command, something is still extending the execution time.

Reply:
With prune command added, execution takes more than 28 seconds.

Reply:
Should be fixed by combining fetch operations into one, instead of potentially 4 separate fetches.